uber/models.py

```diff
@@ -700,13 +700,15 @@ def delete_from_group(self, attendee, group):
             self.delete(attendee)
             group.attendees.remove(attendee)
 
-        def assign_badges(self, group, new_badge_count, new_badge_type=None, **extra_create_args):
-            new_badge_type = new_badge_type or c.ATTENDEE_BADGE
```

Comment:
I (dumbly) didn't realize there were other places where this was getting called.  I wonder if there are situations where we assign_badges() to new badges and then end up creating the wrong type of ribbon or something.